protocol/java/src/dk/itu/epbprtv0/Runner.java

```diff
@@ -0,0 +1,133 @@
+package dk.itu.epbprtv0;
```

Comment:
What's the purpose of all the Java code? I don't think any of the ANN algorithms are written in Java right?

Would like to keep this to Python and C++ – is there a way to remove the Java code?